backend/entities/office_hours/event_entity.py

```diff
@@ -0,0 +1,158 @@
+"""Definition of SQLAlchemy table-backed object mapping entity for Office Hour Sections."""
```

Comment:
Small Nit: I wonder if naming this file `oh_event_entity.py` might be better, only because `event_entity` is another file used in the organizations feature! It might become confusing when importing files - however, feel free to disregard this message too.

Reply:
(Same with `section_entity`, since there is also a section table in the academics feature!)